tests/end2end/features/utilcmds.feature

```diff
@@ -164,3 +164,7 @@ Feature: Miscellaneous utility commands exposed to the user.
     Scenario: Using debug-log-filter with invalid filter
         When I run :debug-log-filter blah
         Then the error "filters: Invalid value blah - expected one of: statusbar, *" should be shown
+
+    Scenario: Using debug-log-filter
+        When I run :debug-log-filter webview
+        Then no crash should happen
```

Comment:
So it looks like `test_*.py` files need to be unique globally across the entire project?

I worked around this by choosing a silly filename to test this hypothesis, but this seems like a strange requirement?

Any advice? I can dig deeper later if you don't know.